### ISSUE TITLE
Issues building the documentation

### ISSUE THREAD

**Issue by gurbraj** [auth0/ruby-auth0#135]
Hello,

When I run 
`$ bundle exec rake documentation `
I get
```
bundle exec rake documentation        
rake aborted!
Don't know how to build task 'documentation' (see --tasks)
```
When I list my tasks using `--tasks`, documentation does not show up.

Any ideas?

When the local documentation is built, will it be the same as what is on https://www.rubydoc.info/github/auth0/ruby-auth0/Auth0Client? It seems quite incomplete...

**Comment by joshcanhelp** [comment]
Sorry for the trouble here and thank you for the report!

This is the up-to-date docs for the latest version:

https://www.rubydoc.info/gems/auth0/4.5.0/Auth0/Api/AuthenticationEndpoints

I'll investigate why the link you posted here is not up-to-date. It mentions `master` on there but I'm not seeing the latest auth endpoint docs added. 

I'm seeing the task on my end and it runs without any issues (a few warning that need cleaning up). It creates an up-to-date version in the `/docs` folder for this gem. [The task is initiated here](https://github.com/auth0/ruby-auth0/blob/master/Rakefile#L16). It might be a problem with how that task is structured so you can try one of the fixes [here](https://stackoverflow.com/questions/5689653/why-is-my-custom-rake-task-in-lib-tasks-not-discovered-in-rails-3). If one of those work, let me know or create a PR and I'll test it out. 

Also, can you try @jaimietanna's fix above (#136) and see if that makes a difference?

Thank you!

**Comment by joshcanhelp** [comment]
Closed for no activity
